src/tracker/index.js

```diff
@@ -10,25 +11,25 @@ const ISSUE_STATE_CLOSED = 'closed';
 const ISSUE_STATE_OPEN = 'open';
 const ISSUE_STATE_ALL = 'all';
 
-const UPDATE_DOCUMENT_LABEL = process.env.GITHUB_LABEL_UPDATE || 'update';
+const UPDATE_DOCUMENT_LABEL = config.get('tracker.githubIssues.label') || 'update';
```

Comment:
If we use a config, the default should be set in config and not here 🙂

Reply:
I agree and I hesitated to remove it but the aim of this PR was not to refactor Tracker module, so I left it as is.